config/vufind/config.ini

```diff
@@ -394,6 +394,7 @@ force_first_scheduled_email = false
 method         = Database
 ;method         = AlmaDatabase
 ;method         = Shibboleth
+;method         = ShibbolethWithWAYF
```

Comment:
Looks like config.ini needs to be updated to reflect the latest changes -- removing this new method and instead adding the override setting.